reconcile/utils/vault.py

```diff
@@ -27,6 +27,10 @@ class SecretNotFound(Exception):
     pass
 
 
+class SecretFormatProblem(Exception):
```

Comment:
```suggestion
class SecretKeyFormatError(Exception):
```
Just a NIT. I think it's more clear

Reply:
👍 will adopt this as part of the code relocation.